dask/dataframe/tests/test_nunique_approx.py

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
Slight preference to either rename this file so that it matches an existing implementation file (e.g. `_hyperloglog.py`), or merge with another file.